CHANGELOG.md

```diff
@@ -39,7 +46,8 @@ IMPROVEMENTS:
 BUG FIXES:
 
 * resource/vcd_vapp, resource/vcd_vapp_vm add vApp status handling when environment is very fast ([#68](https://github.com/terraform-providers/terraform-provider-vcd/issues/68))
-* resource/vcd_vapp_vm add additional validation to check if vApp template is OK [[#157](https://github.com/terraform-providers/terraform-provider-vcd/issues/157)] 
+* resource/vcd_vapp_vm add additional validation to check if vApp template is OK [[#157](https://github.com/terraform-providers/terraform-provider-vcd/issues/157)]
+* Insert Media now supports force ejecting on running VM
```

Comment:
It looks like we have an auto-merge bubble here. Take a look at a few lines above in the IMPROVEMENTS section - we have this feature documented already. Let's better not touch this older version's changelog :)